index.html

```diff
@@ -109,60 +120,66 @@ <h2>Tonic</h2>
             </div>
         </div>
     </section>
-    <section class="about">
+    <section class="about" id="about">
         <div class="about-content">
-            <div class="container">
-                <h2>About myself</h2>
-                <p>Hello I'm a software developer! I can help you build a product,  
-                feature or website. Look through some of my work and experience!
-                If you like what you see and have a project you need coded, don't hesitate 
-                to contact me. 
-                </p>
-                <span>LET'S CONNECT</span>
-                <ul class="social-media">
-                    <li><a href="https://twitter.com/DavidEspinoG"><img src="./img/twitter.png" alt="twitter icon"></a></li>
-                    <li><a href="https://www.linkedin.com/in/davidespinog/"><img src="./img/linkedin.png" alt="linkedin icon"></a></li>
-                    <li><a href="#"><img src="./img/M.png" alt="M icon"></a></li>
-                    <li><a href="https://github.com/DavidEspinoG"><img src="./img/git.png" alt="github icon"></a></li>
-                    <li><a href="#"><img src="./img/hand.png" alt="hand icon"></a></li>
-                </ul>
-                <a><button>Get my resume</button></a>  
-
-                <div class="languages">
-                    <h3>Languages</h3>
-                    <i class="fa-solid fa-chevron-down fa-xl"></i> 
+            <div class="container desktop-about">
+                <div class="about-column">
+                    <h2>About myself</h2>
+                    <p>Hello I'm a software developer! I can help you build a product,  
+                    feature or website. Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, don't hesitate 
+                    to contact me. 
+                    </p>
+                    <span>LET'S CONNECT</span>
+                    <ul class="social-media">
+                        <li><a href="https://twitter.com/DavidEspinoG"><img src="./img/twitter.png" alt="twitter icon"></a></li>
+                        <li><a href="https://www.linkedin.com/in/davidespinog/"><img src="./img/linkedin.png" alt="linkedin icon"></a></li>
+                        <li><a href="#"><img src="./img/M.png" alt="M icon"></a></li>
+                        <li><a href="https://github.com/DavidEspinoG"><img src="./img/git.png" alt="github icon"></a></li>
+                        <li><a href="#"><img src="./img/hand.png" alt="hand icon"></a></li>
+                    </ul>
+                    <a><button>Get my resume</button></a>  
                 </div>
```

Comment:
- [ ] Kindly implement the _**hover**_, _**active**_, _**enabled**_ and _**disable**_ state for all the buttons in your code according to the design on Figma. I noticed that all the buttons do not have this feature. Kindly implement this as it is a requirement in this project. 👍 

#### Hint: You can check details and styles declaration for these button states by checking the Button part of the template on Figma For visual reference, kindly see the screenshot below 💻 

![image](https://user-images.githubusercontent.com/77106962/213263448-85bf3334-bf87-4a82-9704-9117d226cfb2.png)